Source/Core/Common/GekkoDisassembler.cpp

```diff
@@ -2328,28 +2359,30 @@ std::string GekkoDisassembler::Disassemble(u32 opcode, u32 current_instruction_a
   return m_opcode.append("\t").append(m_operands);
 }
 
-static const char* gprnames[] = {
+constexpr std::array<const char*, 32> gpr_names{
```

Comment:
Not sure if thats a GitHub thing, but the only place where this appears is inside `GetGPRName`, and `GekkoDisassembler::GetGPRName` only appears in declaration and definition (no calls) when I search for them.
Same for `GetFPRName`.

I was going to ask if those could be made `constexpr` as well, but since they don't seem to be called I'm not even sure if we should keep them around. They make _sense_ in the context of a disassembler, but why do we even have them if we don't use them?

Reply:
They seem to be unused indeed...  Kind of odd that they're not used in a disassembler?